CONTRIBUTING.md

```diff
@@ -106,3 +106,45 @@ The website sources are located here: [wiremock/wiremock.org](https://github.com
 All the documentation is located under the `_docs` directory as Markdown files,
 and it can be edited with all modern text editors and IDEs.
 See the repository's contributor guide for more information.
+
+## Merge process
+
+Merges to this repository can be performed by the WireMock maintainer ([Tom Akehurst](https://github.com/tomakehurst))
+and by _co-maintainers_ assigned by him.
+This is a [community role](https://github.com/wiremock/community/blob/main/governance/README.md)
+designed for WireMock itself and other key repositories,
+specifically to facilitate review and changes while WireMock 3 is in beta
+and receives a lot of incremental patches.
+
+- The maintainers are responsible to verify the pull request readiness
+  in accordance with contributing guidelines (e.g. code quality, test automation, documentation, etc.).
+  The pull request can only be approved if these requirements are met
+- In the beginning, a review by one co-maintainer is required for the merge,
+  unless there are negative reviews and unaddressed comments by other contributors
+- After the approval, it is generally recommended to give at least 24 hours for reviews before merging
+
+Beyond WireMock 3.x Beta releases, the scope of responsibilities for co-maintainers
+is yet to be determined based on the experiences with this role for the Beta versions.
+
+### What can be merged by co-maintainers
+
+While WireMock 3.x is in Beta, co-maintainers can merge the following pull requests:
+
+- Minor features and improvements that do not impact the WireMock architecture
+- Refactorings, including the major ones, e.g. Guava replacement
+- Test Automation
+- Non-production repository changes: documentation (including Javadoc), GitHub Actions, bots and automation
+- Dependency updates for shaded dependencies, patch/minor versions for projects following the Semantic Versioning notation
+
+### What CANNOT be merged by co-maintainers without BDFL’s approval
+
+The following changes need a review by Tom Akehurst before being merged.
```

Comment:
Although I agree with this - if something was to be merged, it's still possible to revert it before the next release, right? I _assume_ Tom right now is the only person releasing the library, and it's unlikely(?) that folks are generally using SNAPSHOTs or pulling the latest commits, so it may be less worrisome if something did get merged and needs to get unpicked?